src/libraries/Common/src/Interop/Interop.HostPolicy.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static class HostPolicy
+    {
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl, CharSet = CharSet.Auto)]
+        internal delegate void corehost_resolve_component_dependencies_result_fn(string assemblyPaths,
+            string nativeSearchPaths, string resourceSearchPaths);
+
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl, CharSet = CharSet.Auto)]
+        internal delegate void corehost_error_writer_fn(string message);
+
+#pragma warning disable BCL0015 // Disable Pinvoke analyzer errors.
+
+        [DllImport(Libraries.HostPolicy, EntryPoint = "corehost_resolve_component_dependencies",
+            CallingConvention = CallingConvention.Cdecl, CharSet = HostpolicyCharSet)]
+        private static extern int ResolveComponentDependencies(string componentMainAssemblyPath,
```

Comment:
```suggestion
        private static extern int corehost_resolve_component_dependencies(string componentMainAssemblyPath,
```

From https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/interop-guidelines.md#naming : 

> Interop signatures / structs / constants should be defined using the same name / capitalization / etc. that's used in the corresponding native code.